config/management/operational/src/validator_config.rs

```diff
@@ -31,6 +31,42 @@ pub struct SetValidatorConfig {
 }
 
 impl SetValidatorConfig {
+    fn address_valid(network_address: &NetworkAddress) -> Result<(), Error> {
+        // Only allow DNS and IP addresses
```

Comment:
Guess we don't have IPv6 support, right?

Reply:
woops, missed one :)